openshift/templates/nodejs-mongodb.json

```diff
@@ -160,6 +160,10 @@
                 ],
                 "env": [
                   {
+                    "name": "DATABASE_SERVICE_NAME",
```

Comment:
We need this to be able to read the env var `${DATABASE_SERVICE_NAME}_SERVICE_HOST` and `${DATABASE_SERVICE_NAME}_SERVICE_PORT`.